flepimop/gempyor_pkg/src/gempyor/initial_conditions.py

```diff
@@ -177,7 +182,7 @@ def read_initial_condition_from_tidydataframe(
             logger.critical(
                 f"No initial conditions for for subpop {pl}, assuming everyone (n={modinf.subpop_pop[pl_idx]}) in the first metacompartment ({modinf.compartments.compartments['name'].iloc[0]})"
             )
-            raise ValueError("THERE IS A BUG; REPORT THIS MESSAGE. Past implemenation was buggy")
+            raise ValueError(f"THERE IS A BUG; REPORT THIS MESSAGE. Past implemenation was buggy.")
```

Comment:
WHY ARE WE YELLING? Jokes aside the all caps is aggressive and this seems more like a `RuntimeError` then a `ValueError`.